src/kili/services/export/format/voc/__init__.py

```diff
@@ -25,7 +26,9 @@ class VocExporter(AbstractExporter):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.download_media = True  # we need the assets for Pascal VOC annotation format
+        if not self.download_media:
+            warnings.warn("For Pascal VOC export, download_media must be set to True.")
```

Comment:
"For the export to the Pascal VOC format, the download of assets cannot be disabled"
download_media is not the parameter name in the CLI.